docs/auth/README.adoc

```diff
@@ -13,3 +13,118 @@ dependencies {
 
 == Usage
 
+To use the Auth SDK you'll first need to import the Core module. See link:../core/README.adoc[the Core documentation].
```

Comment:
I think it should link to this doc: https://github.com/aerogear/aerogear-android-sdk/blob/master/docs/getting-started.adoc

Reply:
ah, never mind. It is for the core module.

Reply:
I think we need at least 2 prerequisites steps:

1. import the core module.
2. specify the redirect uri. It should be specified in `android.defaultConfig.manifestPlaceholders` of the build.gradle file (see https://github.com/aerogear/aerogear-android-sdk/blob/master/example/build.gradle#L17). This value should also be the one of the "valid redirect URIs" configuration in Keycloak